ui/site/src/assets.ts

```diff
@@ -16,38 +15,30 @@ export const url = (path: string, opts: AssetUrlOpts = {}) => {
 // bump flairs version if a flair is changed only (not added or removed)
 export const flairSrc = (flair: Flair) => url(`flair/img/${flair}.webp`, { version: '_____2' });
 
-const loadedCss = new Map<string, Promise<void>>();
-export const loadCss = (href: string, media?: 'dark' | 'light'): Promise<void> => {
-  if (!loadedCss.has(href)) {
+const loadedCss = new Set<string>();
+export const loadCss = (href: string): Promise<void> =>
+  new Promise(resolve => {
+    if (loadedCss.has(href)) return resolve();
     const el = document.createElement('link');
     el.rel = 'stylesheet';
     el.href = url(site.debug ? `${href}?_=${Date.now()}` : href);
```

Comment:
Could this cache-buster be removed?

Reply:
it's not really a cache buster, but i also doubt there are any performance implications with multiple identical css links in the head tag. also, it would be extremely rare for this to happen. on the other hand, it doesn't hurt anything and it's conceivably faster for the browser to test set inclusion rather than consult its cache.

Reply:
I meant for the query string piece of it

```suggestion
    el.href = url(href);
```